src/status_im/ui/screens/chat/photos.cljs

```diff
@@ -12,7 +12,8 @@
      [react/view {:style (style/photo-container size)}
       [fast-image/fast-image {:source              {:uri photo-path}
                               :style               (style/photo size)
-                              :accessibility-label (or accessibility-label :chat-icon)}]
+                              :accessibility-label (or accessibility-label :chat-icon)
+                              :fallback true}]
```

Comment:
Any reason adds `:fallback` here?

Reply:
i doubt that there r some issues exists with fast-image for the implementation on the ios side which cause CASE 1 reported from #14236 , after add this option, looks like it resolved CASE 1, need feedback from test team .

Reply:
![image](https://user-images.githubusercontent.com/15090582/198611690-c1d4c8f1-d079-4efa-b10c-b40f42efaa29.png)

Seems all the broken avatar are identicon, which is the majority of avatars. We better take a closer look into this, since multiple places are using `fast-image` and we do want it to work for default identicon for performance improvements

Reply:
i'm sure fast-image is the cause of case 1, but i'm not familiar with OC, could u help to check implementation of native? currently what i found is fast-image triggered the callback `on-error`, but there is not much useful info with the callback param @yqrashawn

Reply:
one improvement would be remove the clock param for `get-identicons-uri` (status-im.utils.image-server/get-identicons-uri) , so identicon can be cached. i will try it later.

Reply:
I'd be happy to look into it.

Reply:
another improvement can be `get-account-image-uri`, we need to know when the account image get updated , just like implementation of `get-contact-image-uri`

Reply:
> another improvement is when fast-image failed load image (on-error), we may make it retry again at once

okay, this shouldn't be the option, as images loaded from localhost